sag_et.par

```diff
@@ -4,9 +4,13 @@
 ! s = list of subdivisions for above division using non-native fifth
 !
 n 5 3
+/N\

```

Comment:
Even though 5EDO doesn't use any symbols, a placeholder is necessary here to prevent it from being merged with other notations. With this configuration, Scala still does the right thing and uses bare nominals CDEGA for the native fifth 5EDO.